common/services/person.test.js

```diff
@@ -1,8 +1,18 @@
 const { forEach } = require('lodash')
+const proxyquire = require('proxyquire')
 
+const filters = require('../../config/nunjucks/filters')
 const apiClient = require('../lib/api-client')()
 
-const personService = require('./person')
+const formatDateStub = sinon.stub(filters, 'formatDate').returns('28 Oct 2010')
+
+const unformat = proxyquire('./person-unformat', {
```

Comment:
I wonder if its cleaner to test `person.unformat.js` separately in `person.unformat.test.js` then you don't need to worry about nested proxyquires. 

**BUT** not as part of this PR/change. One to think about if you revisit this during STC/SCH/Prison edit changes.

One option could be to use an index loader pattern that is used elsewhere for the person methods. 

So `person.js` becomes:
```
./services
  ...
  /person
    index.js
    format.js
    format.test.js
    unformat.test.js
    unformat.test.js
    create.js
    create.test.js
    ...
  ...
```

Reply:
Sure. I didn't want to add yet more noise at the moment. I really only split it out in the first place to placate the CodeClimate gods